prow/e2e-suite.sh

```diff
@@ -45,22 +45,41 @@ SINGLE_MODE=false
 # Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
 # for existing resources types
 RESOURCE_TYPE="${RESOURCE_TYPE:-gke-e2e-test}"
-OWNER='e2e-suite'
-INFO_PATH="$(mktemp /tmp/XXXXX.boskos.info)"
-FILE_LOG="$(mktemp /tmp/XXXXX.boskos.log)"
+OWNER="${OWNER:-e2e-suite}"
+PILOT_CLUSTER="${PILOT_CLUSTER:-}"
+USE_MASON_RESOURCE=${USE_MASON_RESOURCE:-True}
+CLEAN_CLUSTERS=${CLEAN_CLUSTERS:-True}
 
-E2E_ARGS=(--mason_info="${INFO_PATH}")
 
 source "${ROOT}/prow/lib.sh"
 source "${ROOT}/prow/mason_lib.sh"
 source "${ROOT}/prow/cluster_lib.sh"
 
 function cleanup() {
-  mason_cleanup
-  cat "${FILE_LOG}"
+  if [[ "$CLEAN_CLUSTERS" == "True" ]]; then
+    unsetup_clusters
```

Comment:
This should be done by the framework.

Reply:
@sebastienvas  -- by "framework" are you meaning Mason?  

The main thing that needs to happen in `unsetup_clusters` is that the firewall rule needs to be removed from the project since Mason won't do that just by destroying clusters.  The other stuff I added in so I could rerun the tests over and over without always deleting and recreating clusters in my own GKE proj.

Reply:
I meant to go e2e framework.

Reply:
The e2e framework won't be able to delete the gcloud firewall rule so something like unsetup_clusters is needed for GKE projs usage.

Also, the service-account stuff was done by cluster_lib.sh and unsetup_clusters removes that from the remote cluster.  There's a PR to put the service account create inside the helm chart used to setup the remote cluster.  But I'd like to get this merged and update the script later along with a fix to the e2e multicluster.go after the helm merges.

Reply:
Sounds good.